templates/docs/examples/patterns/forms/_indeterminate_script.js

```diff
@@ -0,0 +1,33 @@
+/**
+  Sets up indeterminate checkboxes and attaches event handlers to it
+  @param {String} checkboxSelector The CSS selector matching indeterminate checkboxes
+  @param {String} triggerSelector The CSS selector matching the trigger element
+*/
+function initIndeterminateCheckboxes(checkboxSelector, triggerSelector) {
+  var checkbox = document.querySelector(checkboxSelector);
+  var trigger = document.querySelector(triggerSelector);
+
+  // FOR EXAMPLE PURPOSES ONLY:
+  // In this example, we want the indeterminate state to be visible to Percy,
+  // which doesn't use JavaScript, so we hardcode the `aria-checked='mixed'`
+  // attribute in HTML, and remove it when JavaScript _is_ available.
+  checkbox.removeAttribute('aria-checked');
```

Comment:
We want to also make this checkbox indeterminate at the same time:

```suggestion
  checkbox.removeAttribute('aria-checked');
  checkbox.indeterminate = true;
```